.changeset/light-elephants-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Changed: stricter `possible` values for `block-no-empty`, `declaration-property-max-values`, `declaration-property-value-no-unknown` and `number-max-precision`
```

Comment:
[suggest] Sorry for my repeated comments. How about using "Fixed" instead of "Changed" to clarify this PR is a bug fix? In addition, since "possible" seems internal, so I'd like to use "primary" and "secondary" instead for general users.

```suggestion
Fixed: primary or secondary options for `block-no-empty`, `declaration-property-max-values`, `declaration-property-value-no-unknown` and `number-max-precision` to be stricter
```